src/cdi_ndi/NDI_CP_Eloss.cc

```diff
@@ -236,6 +241,10 @@ void NDI_CP_Eloss::load_ndi() {
   for (auto &temperature : temperatures) {
     temperature = exp(temperature);
   }
+
+  //! Close datafile
+  ndi_error = NDI2_close_gendir(gendir_handle);
```

Comment:
Oooooh, good catch!! I wonder if this was part of the problem we saw a while ago where our first driver test was much faster than subsequent tests... In any case, fixed now, woo!

Reply:
We may not have been calling this in our earlier driver tests -- in my experience, not closing the gendir here just causes the next attempt by Draco to read from the NDI gendir file to fail :)

Reply:
Ohhhhh right, good point :)